lib/locomotive/steam/services/site_finder_service.rb

```diff
@@ -6,7 +6,7 @@ class SiteFinderService
       attr_accessor_initialize :repository, :request
 
       def find
-        repository.by_domain(request.host)
+        repository.by_domain(request.host) if request
```

Comment:
No pb! One last question, do you remember what the scenario was in order to reproduce this error? (I'm going to write a spec and then fix it). thanks again!